packages/sources/vue/src/components/overlays/VtmnToast/VtmnToast.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+import '@vtmn/css-toast/dist/index-with-vars.css';
+import { reactive, computed, defineComponent, PropType } from 'vue';
+import { VtmnButton } from '../../index';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnToast',
+  inheritAttrs: false,
+  components: { VtmnButton },
+  props: {
+    withIcon: {
+      type: Boolean,
+      default: false,
+    },
+    content: {
+      type: String,
+      required: true,
+    },
+    withCloseButton: {
+      type: Boolean,
+      default: false,
+    },
+    timeout: {
```

Comment:
I don't think this should be handle inside the component.

You can find here a implementation of toast notifications (home made) based on VTMN CSS classes.
https://github.com/dktunited/finance-front-check-and-go/tree/develop/src/components/notifications

You can also look at the BaseAlert component of this project to better understand what i'm saying.

Feel free to reach me if you want to talk about this !